fwdctl/src/dump.rs

```diff
@@ -91,16 +91,16 @@ pub struct Options {
     pub max_key_count: Option<u32>,
 
     /// The output format of database dump.
-    /// Possible Values: ["csv", "json", "stdout"].
+    /// Possible Values: ["csv", "json", "stdout", "dot"].
     /// Defaults to "stdout"
     #[arg(
         short = 'o',
         long,
         required = false,
         value_name = "OUTPUT_FORMAT",
-        value_parser = ["csv", "json", "stdout"],
```

Comment:
clap has a macro for this

```
#[derive(Debug, clap::ValueEnum)]
enum OutputFormat {
  Csv,
  Json,
  Stdout,
  Dot,
}
```

and replace this `value_parser` with `value_enum`:

```
#[arg(value_enum, ...)]
pub output_format: OutputFormat,
```